sisl/physics/electron.py

```diff
@@ -735,7 +735,12 @@ def berry_phase(contour, sub=None, eigvals=False, closed=True):
     eigvals : bool, optional
        return the eigenvalues of the product of the overlap matrices
     closed : bool, optional
-       whether or not to include the connection of the last and first points in the loop (unwanted for the Zak phase)
+       whether or not to include the connection of the last and first points in the loop
+    method : string, optional
```

Comment:
when string options are in play one should typically do:

`method : {'berry', 'zak'}`

where the first entry is the default value. This is according to the numpy doc-standards.

Reply:
OK, I will change this.